File: SistemaParadaCerta/APIs-ParadaCerta/paradacerta-api/src/main/java/com/paradacerta/api/model/SessaoAtivaResponse.java
package com.paradacerta.api.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class SessaoAtivaResponse {
    private String sessaoId;
    private Integer estacionamentoId;
    private String estacionamentoNome;
    private String pixKey;
    private long horaEntrada;       // Unix timestamp em ms
    private Long inicioReservaPrevisto; // Unix timestamp em ms, apenas reservas
    private BigDecimal precoHora;
    private String placa;
    private String modeloVeiculo;
    private boolean reservado;
}

File: SistemaParadaCerta/AplicacaoMobile-ParadaCerta/ParadaCerta/app/src/main/java/com/example/paradacerta/models/SessaoAtivaResponse.kt
package com.example.paradacerta.models

data class SessaoAtivaResponse(
    val sessaoId: String,
    val estacionamentoId: Int,
    val estacionamentoNome: String,
    val pixKey: String?,
    val horaEntrada: Long,
    val inicioReservaPrevisto: Long? = null,
    val dataHoraConfirmacao: Long? = null,
    val precoHora: Double,
    val placa: String?,
    val modeloVeiculo: String?,
    val reservado: Boolean? = null,
    val status: String? = null,
    val valorPagoAntecipado: Double? = null
)
